src/Sulu/Bundle/AdminBundle/Resources/js/views/List/List.js

```diff
@@ -124,7 +125,9 @@ class List extends React.Component<Props> {
 
         const metadataOptions = this.buildMetadataOptions(
             attributes,
-            routerAttributesToListMetadata
+            props.resourceStore,
```

Comment:
I would try to keep the order similar to the function above, and move the `resourceStore` at the end.